workers/insight_worker/insight_worker.py

```diff
@@ -676,8 +676,10 @@ def update_metrics(self):
                 "cm_defined": True if metric['is_defined'] == 'true' else False,
                 "cm_api_endpoint_repo": metric['endpoint'],
                 "cm_api_endpoint_rg": None,
+                // cm info key is being used twice
```

Comment:
@sgoggins i am confused here the cm_info key is being used multiple times to two different values, what changes would be appropriate here